index.html

```diff
@@ -3,14 +3,24 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>TV Show Project | My Name (My GitHub username)</title>
+    <title>TV Show Project | Tosin (fowobi)</title>
     <link href="style.css" rel="stylesheet" />
     <link rel="icon" href="/favicon.ico" type="image/x-icon" />
   </head>
 
   <body>
     <div id="root">
+      <!-- <div class="search-container">
+        <input type="text" id="search" placeholder="Search episodes...">
+         <span class="search-count"></span>  -->
+        <!-- <p id="search-count"></p> -->
+      <!-- </div> -->
+      <!-- <div class="episodes-container"></div> -->
+      
     </div>
+    
+    
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```

Comment:
Any reason you wish to use jQuery? CYF syllabus is more aimed at vanilla javascript, and later React

Reply:
Noted